apps/web/app/platform/course-materials/page.tsx

```diff
@@ -0,0 +1,18 @@
+import {
+  getCourses,
+  getCourse,
+  getAllCourses,
+  createCourse,
+} from "@/app/lib/actions";
```

Comment:
What does `pnpm run build` give you with this? They're...unused, no?

But from my previous comment, this is where you'd put these methods instead. Additionally, you'd add security safeguards to make sure only **authorized users** can do these mutations/data fetching.